app/test_main.py

```diff
@@ -1 +1,35 @@
-# write your code here
+import pytest
+from unittest import mock
+from app.main import can_access_google_page
+
+
+@pytest.fixture(scope="module")
+def url() -> str:
+    return "https://www.google.com"
+
+
+@pytest.mark.parametrize("internet_connection,"
+                         "valid_url,"
+                         "expected_output",
+                         [
+                             pytest.param(True, True, "Accessible",
+                                          id="internet_available_success"),
+                             pytest.param(False, True, "Not accessible",
+                                          id="internet_unavailable_success"),
+                             pytest.param(True, False, "Not accessible",
+                                          id="internet_available_fail"),
+                             pytest.param(False, False, "Not accessible",
+                                          id="internet_unavailable_fail")
+                         ]
+                         )
```

Comment:
Fix indentation according to this (black)

![image](https://user-images.githubusercontent.com/104218550/221213912-da599b0b-cf60-470d-8b39-0133a8fb50f4.png)

Reply:
if i fix like u, i have a flake8 issue too))))
![image](https://user-images.githubusercontent.com/18147743/221222677-4f999c98-ae1f-4a72-9deb-2f0e3499175b.png)